src/client_side.h

```diff
@@ -367,10 +370,10 @@ class ConnStateData : public Server, public HttpControlMsgSink, private Independ
     /* PROXY protocol functionality */
     bool proxyProtocolValidateClient();
     bool parseProxyProtocolHeader();
-    bool parseProxy1p0();
-    bool parseProxy2p0();
     bool proxyProtocolError(const char *reason);
 
+    ProxyProtocol::Two::Message::Pointer theProxyProtocolV2Message;
```

Comment:
We should document new methods and new data members. Documenting this will become easier if Message scope concerns are addressed. Right now, theProxyProtocolV2Message may be nil for successfully parsed v2 messages, which is rather weird.